wicket-orientdb/src/main/java/ru/ydn/wicket/wicketorientdb/model/OQueryDataProvider.java

```diff
@@ -63,19 +64,6 @@ public OQueryDataProvider(OQueryModel<K> model)
         model.setParameter(paramName, value);
         return this;
     }
-    
-    /**
-     * Set value for context variable
-     * @param varName name of the variable to set
-     * @param value {@link IModel} for the variable value
-     * @return this {@link OQueryDataProvider}
-     */
-	public OQueryDataProvider<K> setContextVariable(String varName, IModel<?> value)
```

Comment:
Why this method is being deleted? How it's expected to set context variables in this case?